<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_param</class>
 <widget class="QWidget" name="form_param">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>参数号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_param_number">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QSpinBox::up-button, QSpinBox::down-button {
    width: 14px;
    height: 8px;
    border: 1px solid gray;
    background: lightgray;
}
QSpinBox::up-button {
    subcontrol-position: top right;
    border-bottom: none;
}
QSpinBox::down-button {
    subcontrol-position: bottom right;
    border-top: none;
}
QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 5px;
    height: 5px;
}</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>31</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>参数值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spb_param_value">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QSpinBox::up-button, QSpinBox::down-button {
    width: 14px;
    height: 8px;
    border: 1px solid gray;
    background: lightgray;
}
QSpinBox::up-button {
    subcontrol-position: top right;
    border-bottom: none;
}
QSpinBox::down-button {
    subcontrol-position: bottom right;
    border-top: none;
}
QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 5px;
    height: 5px;
}</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ptn_get_param">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ptn_set_param">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>写入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>